zerver/tornado/handlers.py

```diff
@@ -143,7 +144,10 @@ async def write_django_response_as_tornado_response(self, response: HttpResponse
         self.write(response.content)
 
         # Close the connection.
-        await self.finish()
+        try:
+            await self.finish()
+        except StreamClosedError:
```

Comment:
Can you add a comment for why it's correct to ignore this exception? Otherwise lgtm.

Reply:
How’s this?